docs/tutorial.md

```diff
@@ -0,0 +1,182 @@
+# Oria Operator Tutorial
+
+An in-depth walkthrough of building and running a oria-operator with memcached-operator.
+
+## Prerequisites
+
+- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.21.0 or newer
+- User authorized with `cluster-admin` permissions so that you can install an operator, CRDs and change rbac.
+- [GNU Make](https://www.gnu.org/software/make/)
+
+## Overview
+
+We will create a sample project to let you know how it works and this sample will:
+
+- Create a `memcached-operator` using below mentioned steps. For more infomation refer to this [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/)
+- Generate the manifests using `config/default`
+- Comment out `ClusterRole` and `ClusterRoleBinding`. 
+- Create a `ClusterRole` template in a `ScopeTemplate` and associate it with a `ScopeInstance`
+- Run the operator, the `memcached-operator` will start complaining about `RBAC`.
+- Run the `oria-operator`, it will create `ClusterRole` and `ClusterRoleBinding`. This will make sure that `memcached-operator` will not complain about `RBAC`. 
+
+### Install and run oria-operator
+
+Clone the [github repo](https://github.com/operator-framework/oria-operator.git) and  run the below commands on the root of the directory.
+
+```
+$ make install
+
+$ make run
+```
+
+### Create a new project
+
+Use the [Operator SDK](https://sdk.operatorframework.io/docs/installation/) CLI to create a new memcached-operator from the [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/).
+
+
+```sh
+mkdir -p $HOME/projects/memcached-operator
+cd $HOME/projects/memcached-operator
+# we'll use a domain of example.com
+# so all API groups will be <group>.example.com
+operator-sdk init --domain example.com --repo github.com/example/memcached-operator
+```
+
+### Create a new API and Controller 
+
+```sh
+$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached --resource --controller
+Writing scaffold for you to edit...
+api/v1alpha1/memcached_types.go
+controllers/memcached_controller.go
+...
+```
```

Comment:
I would remove the sample output from this code block and only include the command

Reply:
I followed the same tutorial and wanted to sync it whatever we have with golang tutorial just to avoid confusion.

Reply:
removed it

Reply:
I understand trying to keep it in sync with the golang tutorial, but I think for this case we are trying not to reiterate more than necessary since ideally they will use the SDK golang tutorial for more detailed information. 

I am fine with leaving it this way if you feel it is best, but in a scenario like this I personally prefer providing only the information they need to get to the state we want them in

Reply:
agree and I removed that extra content.